apps/onboarding/src/wallet/wallet.service.ts

```diff
@@ -44,7 +38,9 @@ export interface MetaTxMetadata {
   methodId: string
 }
 
-@Injectable()
+@Injectable({
```

Comment:
can you provide a brief comment why we do this and maybe a link to this doc https://docs.nestjs.com/fundamentals/injection-scopes

Reply:
Where would you suggest adding that comment because it goes like this:

RELAYER_SERVICE provider (defined in AppModule) has to be Request scoped which causes:
   RelayerProxyService to be request scoped which causes:
      WalletService to be request scoped with causes:
          AppController to be request scoped

So adding that comment in the WalletService doesn't seam like the obvious choice. Or maybe I should add it everywhere pointing to the dependency that's Request scoped?